common/protob/messages.proto

```
@@ -572,5 +574,7 @@ enum MessageType {
 
     //onekey
     MessageType_DeviceEraseSector= 10026 [deprecated = true];
+    MessageType_UnLockDevice = 10030 [(bitcoin_only) = true, (wire_in) = true];
```

Buf (1.55.1)

Enum value name "MessageType_UnLockDevice" should be UPPER_SNAKE_CASE, such as "MESSAGE_TYPE_UN_LOCK_DEVICE".

(ENUM_VALUE_UPPER_SNAKE_CASE)